scripts/build.js

```diff
@@ -11,4 +11,4 @@ tasks.copyAssets('build', argv.env);
 console.log('[Webpack Build]');
 console.log('-'.repeat(80));
 
-exec(`./node_modules/.bin/webpack --config webpack/${argv.env}.config.js --progress --profile --colors`);
+exec(`./node_modules/.bin/webpack --config webpack/prodConfig.js --progress --profile --colors --env=${argv.env}`);
```

Comment:
So I don't get it, why we always use `prodConfig`? Even when env might be `dev`

Reply:
the `build.js` is purely for production builds. 

The `dev` environment has absolutely nothing to do with `development` builds. Having a network type `dev` was all a hack used in Icarus to get it to load up the right manifest file that allows connection to localhost.